doc/deps.md

```diff
@@ -143,7 +147,7 @@ download and recompile ODB using a new C++ compiler.
 wget http://www.codesynthesis.com/download/odb/2.4/libodb-2.4.0.tar.gz
 tar -xvf libodb-2.4.0.tar.gz
 cd libodb-2.4.0
-./configure --prefix=<odb_install_dir>
+./configure --prefix=<libodb_install_dir>
```

Comment:
These **do not** need to go into separate folders. It'd be much better if all the 3rd-party dependencies went into the same folder (e.g. Clang, Thrift and ODB into the same "virtual root prefix")... This will create even more confusion, as ODB is something that is integrated together and should exist together (so when you need to upgrade or uninstall or change something you don't have to hunt down 7 different folders), I really don't agree with this change...